zips/draft-ecc-lockbox-disbursement.md

```diff
@@ -0,0 +1,245 @@
+  resulting funds in the form of development grants, according to the rules set
+  by either [^draft-ecc-community-and-coinholder] or [^draft-ecc-zbloc], or
+  another similar proposal.
+* Extension of protocol-based development funding blocks starting from the
+  scheduled end height of the current ``FS_DEFERRED`` and ``FS_FPF_ZCG``
+  funding streams defined in ZIP 1015 [^zip-1015-funding-streams].
+* A variety of different mechanisms that may be used for distributing funds
+  that accrue during the period of the extension, which may vary depending upon
+  which proposal this mechanism is used.
+
+# Requirements
+
+* Funds are held in a multisig resistant to compromise of some of the parties'
+  keys, up to a given threshold.
+* No single party's non-cooperation or loss of keys is able to cause any
+  Protocol-defined Ecosystem Funding to be locked up unrecoverably.
+* The funds previously accrued the Deferred Dev Fund Lockbox as the activation
+  height of this ZIP will be usable immediately on activation.
+
+# Specification
+
+This ZIP proposes the creation of a new Zcash Development Fund. The balance of
+this Fund consists of the contents of the ZIP 1015 Deferred Development Fund
+Lockbox as of the activation height of this ZIP, plus any funds that later
+accrue to either the lockbox or to one or more transparent multisig addresses
+as specified by this ZIP.
+
+### One-time lockbox disbursement
+
+The coinbase transaction of the activation block of this ZIP MUST include an
+additional output to a 3-of-5 P2SH multisig with keys held by the following
+"Key-Holder Organizations": Zcash Foundation, the Electric Coin Company,
+Shielded Labs, and two other organizations yet to be decided.
+
+Let $v$ be the zatoshi amount in the Deferred Dev Fund Lockbox at the
+activation height. ($v$ can be predicted in advance given that height.)
+
+The additional coinbase output MUST follow the same consensus rules as apply to
+funding stream outputs [^zip-0207-consensus-rules]. That is, the coinbase
+transaction MUST contain at least one output that pays $v$ zatoshi, in the
+prescribed way defined in ZIP 207, to the above P2SH multisig address. $v$
+zatoshi are added to the transparent transaction value pool to fund this
+output, and subtracted from the balance of the Deferred Dev Fund Lockbox (i.e.
+the latter balance is reset to zero).
+
+Exactly one of the following options will also be taken. The proposal that
+activates this ZIP must define values for the following two parameters:
+
+* $\mathsf{stream\_value}$: the percentage of the block subsidy to send to 
+  a new funding stream, as described in the options below.
+* $\mathsf{stream\_end\_height}$: The ending block height of that stream.
+
+### Option 1: Extend the lockbox funding stream
+
+The ``FS_DEFERRED`` lockbox funding stream is set to receive
+$\mathsf{stream\_value}\%$ of the block subsidy and is extended until block
+height $\mathsf{stream\_end\_height}$ Both of these parameters must must be
```

Comment:
Missing ".".

Reply:
Addressed in https://github.com/zcash/zips/pull/994